SharpSite.Web/PluginManager.cs

```diff
@@ -131,7 +131,6 @@ public static Task<ApplicationState> LoadPluginsAtStartup(ApplicationState state
 		{
 			// create the plugins folder if it doesn't exist
 			Directory.CreateDirectory( "plugins");
-			Directory.CreateDirectory( "plugins/_wwwroot");
```

Comment:
Let's just move both of these lines up, and always run them at startup, and no further check is needed.